Sources/Core/API/Actor.swift

```diff
@@ -170,6 +170,17 @@ public final class Actor: SceneObject, InstanceHashable, ActionPerformer,
         layer.removeFromSuperlayer()
     }
 
+    /// Check if it's in contact with an Actor
+    public func isInContact(with actor: Actor) -> Bool {
+        print(self.actorsInContact)
+        return self.actorsInContact.contains(actor)
```

Comment:
Looks really good! 👍
I think you can remove the `print` so that it doesn't decrease performance.
Also, you could remove `self.` to be consistent with how properties are accessed in the rest of the code.